Lib/test/test_regions_dictobject.py

```diff
@@ -0,0 +1,50 @@
+import unittest
+
+class TestRegionsDictObject(unittest.TestCase):
+    def setUp(self):
+        enableinvariant()
+
+    def test_dict_insert_empty_dict(self):
+        # Create Region with Empty dictionary
+        r = Region()
+        d = {}
+        r.add_object(d)
```

Comment:
Small comment:

Direct field assignments also work for regions. I think it would be better to use those, since they'll stay while I believe we'll remove `add_object`. This would also "evict" the object, if something triggers the `try_close()` function, while fields will remain.

Reply:
Excellent.  Thanks.  Have updated locally.  Just working on the tests at the moment to get better coverage.